DeAceroDLabs/marketplace-frontend
# [DMP-660] Display Product Price section

## What is done

[[DMP-660] Mostrar Precio de SKU](https://deacero.atlassian.net/browse/DMP-660)

## How is done

1. Added the definition on the `api` and `api.type` files for the prices service.
2. Created a component for handling the price of the product as a child of the ProductDetailView.
3. Connected the ProductPrice component with the new endpoint for getting prices.

## How to test
**Note:** This is not working correctly as the Price service isn't working correctly. We need that the BE and Roobyn include a new param in the productDetail response (we need to get the claArticulo from the articles detail). That said, we can't test if the connection is working correctly, but we can check that the calls and code are implemented correctly. You can try the prices service using some of the next articles keys. If you use the SKU from the productDetail it won't work.

1. On the home screen or the search screen select a product.
2. You should be redirected to the product detail view.
3. Here you will see the new section dedicated to displaying the price, it should be $0.
4. Check on the console, that when a sku is selected, a request to the prices services is sent.

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
